src/test/resources/large/funcotator/funcotator_dataSources/cancer_gene_census/hg38

```diff
@@ -0,0 +1 @@
+hg19/
```

Comment:
I may be misreading this, but did you put the hg19 dir in the hg38 dir?

Reply:
I just checked and it looks fine to me.